scripts/tad_util/build_snp_list.R

```diff
@@ -0,0 +1,63 @@
+# 2016 Gregory Way
+# scripts/tad_util/build_snp_list.R
+
+# Description: 
+# Maps an input file of SNPs (rsIDs) to their respective genomic coordinates
+# based on GRCh37 and dbSNP141 and the ENSEMBL biomaRt bioconductor package.
+# The input SNP file may have multiple columns but each column MUST have a
+# column name. The input SNP file also must be comma separated (.csv).
+
+# Usage:
+# The script is run in the command line
+
+#           Rscript --vanilla scripts/tad_util/build_snp_list.R
+
+# and takes two positional arguments as input:
+
+#         --snp_file        <LOCATION_OF_SNP.csv>
+#         --output_file     <MAPPING_FILE.tsv>
+
+# Output:
+# A tab separated file of rsIDs mapped to genomic coordinates
+
+library("checkpoint")
+suppressMessages(checkpoint("2016-02-25"))
+
+library(biomaRt)
```

Comment:
I think it would be helpful if you use `library` import and `namespace` in a more consistent manner. In this script, I saw both `dplyr::filter` and `filter`. Also, you imported `biomaRt` and `dplyr` at the beginning, but directly used `readr` in the script.